docs/testimonials.md

```diff
@@ -0,0 +1,92 @@
+you in our favor!
+
+## What the users have to say 
+
+> I've spent many years of my career working on version control. What I like most
+> about Jujutsu is how it has non-obvious solutions to UX problems that we've run
+> into in the past. Most people may not realize this at first, but the many novel
+> features in Jujutsu all interlock to make it easy to use.
+> 
+> For example, consider Jujutsu's support for automatically rebasing descendants 
+> of amended revisions. When we implemented that in Mercurial, we ran into an 
+> issue: what if there's a merge conflict? Our solution was to warn users if that
+> occurred, and just not perform the auto-rebase. Now, suddenly, users have to 
+> understand that there can be old versions of the same revision visible in their
+> log. They also need to learn how to get themselves out of this state.
+> 
+> In contrast, Jujutsu's solution is to simply make merge conflicts first-class.
+> This is not just an improvement in general, it is also specifically an
+> improvement for auto-rebase—users no longer have to learn about old 
+> versions of a revision unless they want to look at the obslog.
+> 
+> Over and over, I'm struck by how well Jujutsu demonstrates this kind of 
+> evolved thinking, As an experienced version control developer, 
+> I deeply appreciate it.
+
+Rain, engineer at Oxide and former VCS developer
+
+> Jujutsu is pretty cool, you can even keep most of your existing workflows 
+
+Ben, who really doesn't want you keeping your existing workflow.
+
+> Wait, it's not called Jujitsu?
+
+Phil, a Mercurial contributor and user.
+
+> It's the easiest time I've ever had learning a tool this deeply this quickly, 
+> because of the ability to experiment and undo, instead of triple-checking 
+> before trying a new scary command.
+
+Scott Olson, a pretty advanced Git user who replaced it completely with 
+Jujutsu.
+
+> Jujutsu is amazing because it trivializes some of the things I couldn't have come 
+> up with anything remotely as elegant. 
+> It's so rare that a solution attacks the innermost core of a problem so 
+> thoroughly, I genuinely feel blessed to be in its presence. And also a bit 
+> vindicated in not even trying to learn to use any of the tools that felt like
+> more crutches stacked upon a sand castle  
+
+An anonymous user.
+
+> only breaks half as often as my other tools!
+
+jyn, ex-rustc bootstrap developer, a place where tools break down all the time.
+
+
+## What the developers have to say 
+
+> I'm surprised when it works
+
+Martin von Zweigbergk, project lead who contributed over 100k lines of code.
+
+> I'm sometimes still surprised that navigating with `next` and `prev` works.
+
+Philip Metzger, contributor, author of `jj next` and `jj prev`. 
+
+> The phone number you have dialed is no longer in service. Please hang up and try a new number.
+
+Austin Seipp, contributor.
+
+> As a fan of big green checkmarks in the log, made it so that Github is a ppure dopamine hit. 
```

Comment:
```suggestion
> As a fan of big green checkmarks in the log, jj made it so that Github is a pure dopamine hit. 
```

Reply:
That's a great suggestion, thanks. I've only replace `jj` with Jujutsu.